science/pdb2pqr/Portfile

```diff
@@ -1,67 +1,50 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           python 1.0
+PortSystem              1.0
+PortGroup               python 1.0
+PortGroup               github 1.0
 
-name                pdb2pqr
-version             2.1.1
-revision            1
-categories          science
-maintainers         {gmail.com:howarth.at.macports @jwhowarth}
-description         automate Poisson-Boltzmann electrostatics calculations
-long_description    PDB2PQR is a Python software package that automates many of the \
-                    common tasks of preparing structures for continuum electrostatics \
-                    calculations, providing a platform-independent utility for converting \
-                    protein files in PDB format to PQR format.
-homepage            http://pdb2pqr.sourceforge.net/
-platforms           darwin
-master_sites        sourceforge:project/pdb2pqr/pdb2pqr/pdb2pqr-${version}/
-distfiles           ${name}-src-${version}.tar.gz
-checksums           rmd160  b08fc9118cc3b96f57af510829e58af14bbefe16 \
-                    sha256  c2b14b0c95ffd76c910af108541a34beaea5f5c2e44246a61640644b2990c3cb \
-                    size    14711417
+name                    pdb2pqr
```

Comment:
this is already set from the `github.setup` line, please remove (as in the other port ideally you'd move the `github.setup` and associated lines to the top here)